cpp/ql/src/semmle/code/cpp/internal/QualifiedName.qll

```diff
@@ -0,0 +1,343 @@
+class Namespace extends @namespace {
+  string toString() { result = "QualifiedName Namespace" }
+
+  string getName() { namespaces(this, result) }
```

Comment:
Should this be `namespaces(underlyingElement(this),result)` as it is in `Namespace.qll`?

Reply:
No, because `underlyingElement` is defined higher up in the stack of abstractions.